charts/firehose/templ.yaml

```
@@ -0,0 +1,1125 @@
+    app.kubernetes.io/name: firehose
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/version: "v2.0.0-rc.1-geth-v1.13.4-fh2.3-1"
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/component: reader
+    pod: release-name-firehose-reader-0
+    type: p2p # this label is used by the initContainer to select this service
+spec:
+  type: NodePort
+  externalTrafficPolicy: Local
+  ports:
+    - name: tcp-p2p
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)